### DISCUSSION TITLE
how to activate French

### DISCUSSION THREAD

**Discussion by Chouaib99** [hargata/lubelog#507] [Q&A] [answered]
Hello,

I'm trying to install lubelogger in French without success. 

Here my docker compose, can anyone help ?

version: "3.4"
services:
  app:
    image: ghcr.io/hargata/lubelogger:latest
    container_name: Lubelogger
    build: .
    restart: unless-stopped
    # volumes used to keep data persistent
    environment:
      - LC_ALL=fr_FR.UTF-8
      - LANG=fr_FR.UTF-8
      - LOGGING__LOGLEVEL__DEFAULT=Error
    volumes:
      - /home/lubelogger/config:/App/config
      - /home/lubelogger/data:/App/data
      - /home/lubelogger/translations:/App/wwwroot/translations
      - /home/lubelogger/documents:/App/wwwroot/documents
      - /home/lubelogger/images:/App/wwwroot/images
      - /home/lubelogger/temp:/App/wwwroot/temp
      - /home/lubelogger/log:/App/log
      - /home/lubelogger/keys:/root/.aspnet/DataProtection-Keys
    ports:
      - 8887:8080

**Comment by Chouaib99** [comment]
Precision: I would like to use km end litre

**Reply by hargata** [reply]
In the settings uncheck "Use MPG" and your fuel units will now be in liters and your distance unit will be in km.

**Comment by Coldness00** [comment]
Hello, same issue here, how do we set this up?
I do not find anything in the docs?

Thank you in advance

**Reply by hargata** [reply, marked as answer]
For units, In the settings uncheck "Use MPG" and your fuel units will now be in liters and your distance unit will be in km.

For number formats/date formats, make sure you use fr_FR for LC_ALL and LANG environment variables.

For French language translation, download [this file](https://github.com/hargata/lubelog_translations/blob/main/Europe/fr.json) and upload it to LubeLogger using [this guide](https://docs.lubelogger.com/Translations)
